### ISSUE TITLE
Establish a code of conduct

### ISSUE THREAD

**Issue by ansasaki** [keylime/rust-keylime#737]
We should establish a code of conduct for the project/repo.

Should we adopt the same code of conduct from the keylime/keylime repo [1]?

[1] https://github.com/keylime/keylime/blob/master/CODE_OF_CONDUCT.md

**Comment by aplanas** [comment]
I vote for this too, to share the same CoC

**Comment by THS-on** [comment]
Yeah I think it makes sense to share the some CoC and security guidelines between all our repositories. There are three potential options for this:
- Copy it into every our repositories (easily findable, but duplication)
- Link to the documents in the keylime repo
- Create a separate repo that includes all the documents that apply to Keylime

**Comment by ansasaki** [comment]
I think the best here is to add default CoC and other health files like the description in this page:
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

This would make repositories under the `keylime` organization to point to the organization default unless the repository override it.
